examples/cli/src/main/java/io/unitycatalog/cli/delta/DeltaKernelUtils.java

```diff
@@ -54,22 +57,24 @@ public static String createDeltaTable(
       // create an empty table
       TransactionCommitResult commitResult = txn.commit(engine, CloseableIterable.emptyIterable());
       if (commitResult.getVersion() >= 0) {
-        System.out.println("Table created successfully at: " + tablePath);
+        LOGGER.info("Table created successfully at: {}", tablePathUri);
       } else {
         throw new RuntimeException("Table creation failed");
       }
     } catch (Exception e) {
-      throw new IllegalArgumentException("Failed to create delta table", e);
+      String errorMsg =
+          String.format("Failed to create Delta table at '%s': %s", tablePath, e.getMessage());
+      LOGGER.error(errorMsg, e);
```

Comment:
As we will finally throw the exception as root cause to the upper layer,  do we still need to log this redudant stacktrace via logger ?  I will assume it's not neccessary.